contracts/dev-utils/contracts/src/OrderTransferSimulationUtils.sol

```diff
@@ -82,13 +82,13 @@ contract OrderTransferSimulationUtils is
         // Transfer `makerAsset` from maker to taker
         assetData[0] = order.makerAssetData;
         fromAddresses[0] = order.makerAddress;
-        toAddresses[0] = takerAddress;
+        toAddresses[0] = takerAddress == address(0) ? address(1) : takerAddress;
```

Comment:
I feel like we should us a random address instead of `address(1)`. While there is a low probability of `address(1)` being special cased for anything, it is almost infinitely more likely than any random address. 

We should also assign it to a constant that describes its purpose.